framework/Models/PostProcessors/DataClassifier.py

```diff
@@ -1,195 +0,0 @@
-# Copyright 2017 Battelle Energy Alliance, LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-"""
-Created on Jan 29, 2018
-
-@author: Congjian Wang
-"""
-
-import numpy as np
-
-from utils import InputData, InputTypes, utils
-from PluginBaseClasses.PostProcessorPluginBase import PostProcessorPluginBase
-
-class DataClassifier(PostProcessorPluginBase):
-  """
-    This Post-Processor performs data classification based on given classifier.
```

Comment:
are you sure you want to move this out of RAVEN? 
Cant' be this considered a basic RAVEN capabilities (dataset classification)? @wangcj05 @mandd

Reply:
This PP is trying to classify data objects generated by dynamic PRA using information from the data objects generated by static PRA. We may have some niche application in the future, but I think most applications will just use the existing classification algorithms. It seems to me it makes more sense to move it SR2ML. @mandd @alfoa

Reply:
After discussion with Congjian, it makes to move it (maybe I would suggest a change of the name since the user might think this PP performs a data classification (i.e. ROM) based on already classified data...something like Train ROM classifier + custom sampler based on the second dataobject to classify)